docs/data/date-pickers/date-range-picker/CustomInputs.tsx

```diff
@@ -0,0 +1,35 @@
+import * as React from 'react';
+import dayjs from 'dayjs';
+import Box from '@mui/material/Box';
+import { TextFieldProps } from '@mui/material/TextField';
+import { AdapterDayjs } from '@mui/x-date-pickers/AdapterDayjs';
+import { LocalizationProvider } from '@mui/x-date-pickers/LocalizationProvider';
+import { Unstable_NextDateRangePicker as NextDateRangePicker } from '@mui/x-date-pickers-pro/NextDateRangePicker';
+
+type BrowserInputProps = TextFieldProps & {
+  ownerState?: any;
+};
+
+function BrowserInput(props: BrowserInputProps) {
+  const { inputProps, InputProps, ownerState, ...other } = props;
+
+  return (
+    <Box sx={{ display: 'flex', alignItems: 'center' }}>
+      <input {...inputProps} {...(other as any)} />
+      {InputProps?.endAdornment}
+    </Box>
+  );
+}
+
+export default function CustomInputs() {
```

Comment:
Maybe adding ignore at the top of such demo components would stop proptypes from being generated? 🤔 
```tsx
/**
 * @ignore - internal component.
 */
```

Reply:
Unfortunately it does not
We can improve the generation script in a follow up PR :+1:

Reply:
Created #6987 from this. 😉